spec/services/login/after_login_actions_spec.rb

```diff
@@ -66,8 +66,8 @@
 
     context 'saving account_login_stats' do
       let(:user) { create(:user) }
-      let(:login_type) { SAML::User::MHV_ORIGINAL_CSID }
-      let(:login_type_stat) { SAML::User::MHV_MAPPED_CSID }
+      let(:login_type) { SAML::User::MHV_CSID }
+      let(:login_type_stat) { Accountable::MHV_MAPPED_CSID }
```

Comment:
Just a quick fix, I just noticed this, can you make this change real fast?

```suggestion
      let(:login_type_stat) { AccountLoginStatisticsJob::MHV_MAPPED_CSID }
```